index.html

```diff
@@ -14,134 +14,160 @@
-			<h2 class="section-title">Tonic</h2>
-			<div class="section-subtitle">
-				<h6 class="canopy">Canopy</h6>
-				<img src="images/Counter.svg" alt=".">
-				<h6>Back End Dev</h6>
-				<img src="images/Counter.svg" alt=".">
-				<h6>2015</h6>
+			<img class="tonic-section-image mobile" src="images/Snapshoot-Portfolio1.svg" alt="tonic-section-image">
+			<img class="tonic-section-image desktop" src="images/Snapshoot-Portfolio-dt1.svg" alt="tonic-section-image">
+			<div class="tonic-section-description">
+				<h2 class="section-title">Tonic</h2>
+				<div class="section-subtitle">
+					<h6 class="canopy">Canopy</h6>
+					<img src="images/Counter.svg" alt=".">
+					<h6>Back End Dev</h6>
+					<img src="images/Counter.svg" alt=".">
+					<h6>2015</h6>
+				</div>
+				<p class="section-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="technologies">
+					<li>html</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button class="button" type="button">See Project</button>
 			</div>
-			<p class="section-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-			<ul class="technologies">
-				<li>html</li>
-				<li>css</li>
-				<li>javaScript</li>
-			</ul>
-			<button class="button" type="button">See Project</button>
 		</section>
 
-		<section class="tonic-section">
-			<img src="images/Snapshoot-Portfolio2.svg" alt="tonic-section-image">
-			<h2 class="section-title">Multi-Post Stories</h2>
-			<div class="section-subtitle">
-				<h6 class="canopy">Canopy</h6>
-				<img src="images/Counter.svg" alt=".">
-				<h6>Back End Dev</h6>
-				<img src="images/Counter.svg" alt=".">
-				<h6>2015</h6>
+		<section class="tonic-section reverse">
+			<img class="tonic-section-image mobile" src="images/Snapshoot-Portfolio2.svg" alt="tonic-section-image">
+			<img class="tonic-section-image desktop" src="images/Snapshoot-Portfolio-dt2.svg" alt="tonic-section-image">
+			<div class="tonic-section-description">
+				<h2 class="section-title">Multi-Post Stories</h2>
+				<div class="section-subtitle">
+					<h6 class="canopy">facebook</h6>
+					<img src="images/Counter.svg" alt=".">
+					<h6>Full Stack Dev</h6>
+					<img src="images/Counter.svg" alt=".">
+					<h6>2015</h6>
+				</div>
+				<p class="section-description">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+				<ul class="technologies">
+					<li>html</li>
+					<li>Ruby On Rails</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button class="button" type="button">See Project</button>
 			</div>
-			<p class="section-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-			<ul class="technologies">
-				<li>html</li>
-				<li>css</li>
-				<li>javaScript</li>
-			</ul>
```

Comment:
- [ ] Kindly use an indentation of 2 spaces in all your document for readability. This is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and a convention.

Reply:
Dear @mgmediaweb and @Kamba56 
Please can I know on which line is the issue?
In my code, I have 2 spaces indentations everywhere as I corrected the 4 lines. Please help me solve it before I run out of reviews. Thanks

Reply:
Kindly, all over your site within Github there is excessive indentation
![image](https://user-images.githubusercontent.com/41912912/186920344-e0dec1a7-6193-43c4-8c0d-39e797ed7978.png)

In this example I show you how it should look
![image](https://user-images.githubusercontent.com/41912912/186920736-aaf4a29c-dc47-4ef1-887a-f5fd3f1d929f.png)

Reply:
@mgmediaweb 
Is there a way to change that in the GitHub code, please? This is what it looks like in my code in VSCode.

![code-capture](https://user-images.githubusercontent.com/29674658/186927812-7df4bd20-c22f-45db-9d27-48b213a309cd.PNG)

Reply:
Please save it and push the changes to your GitHub.